src/api/app/assets/javascripts/webui2/attributes.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function() {
+  // TODO: Edge case - When a user is already on the page and does a refresh, the description flickers as it appears. Find a way to fix this...
```

Comment:
[nitpick] _Find a way to fix this..._ <- Isn't this redundant with the `TODO`? :rofl: